warp-dba/src/main/java/com/schibsted/nmp/warp/brands/dba/DbaColors.kt

```diff
@@ -197,6 +198,12 @@ internal object DbaComponentColors : WarpComponentColors {
     override val navBar: WarpNavBarColors = DbaNavBarColors
     override val tooltip: WarpTooltipColors = DbaTooltipColors
     override val switch: WarpSwitchColors = DbaSwitchColors
+    override val spinner: WarpSpinnerColors = DbaSpinnerColors
+}
+
+private object DbaSpinnerColors : WarpSpinnerColors {
```

Comment:
No need to create an object for these as the colors used are not custom component tokens. The spinner colors are defined in https://docs.google.com/spreadsheets/d/1Q-Tr_dwJVfxgh3527IFjXQKqtnPPM42QeIr-M6q-zu8/edit?gid=1588196525#gid=1588196525